package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nib-styles/sticky-footer",
-  "version": "0.1.2",
+  "version": "0.2.0",
```

Comment:
This would have been a breaking change (major version bump) as this stands currently as the old classes would no longer work. Now that we are renaming the component completely it will be a new npm package @nib-components/grow-layout and can start the version again at `0.1.0`